src/components/elements/CommonMenuButton.tsx

```diff
@@ -102,6 +102,9 @@ const CommonMenuButton = ({
           vertical: 'top',
           horizontal: 'right',
         }}
+        // Bug: Opening the CommonMenu applies padding to the body, which can look weird on mobile.
```

Comment:
I noticed a visual bug where the 3-dots menu causes the table width to jump around, on mobile. It's also observable on qa, for example [here](https://qa-hmis.openpath.host/projects/==T3FGUzVOaDAxdmc5aWJDOVcvbnB4dz09/enrollments). I'd propose to either open another ticket, or fix it separately (maybe as part of [this pr](https://github.com/greenriver/hmis-frontend/pull/1021))

Reply:
Good find! agreed to fixing separately

Reply:
Opened https://github.com/open-path/Green-River/issues/7220